feature-libs/cart/import-export/core/model/import-export.model.ts

```diff
@@ -3,3 +3,10 @@ export type InvalidFileInfo = {
   invalidExtension?: Boolean;
   fileEmpty?: Boolean;
 };
+
+export type FileValidity = {
+  // size unit is MB
+  maxSize?: Number;
```

Comment:
Let's change it to /** JsDoc comment, so customers can see the description.
Btw. it's good practice to document every configuration property.